blocks/api/categories.js

```diff
@@ -5,7 +5,7 @@
  * @var {Array} categories
  */
 const categories = [
-	{ slug: 'common', title: 'Common' }
+	{ slug: 'common', title: 'Common blocks' }
```

Comment:
Not necessarily in scope of this pull request, but the string ought to be translated.